### ISSUE TITLE
libgap of Group with no generator fails

### ISSUE THREAD

**Issue by fchapoton** [sagemath/sage#35508]
### Is there an existing issue for this?

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.

### Did you read the documentation and troubleshoot guide?

- [X] I have read the documentation and troubleshoot guide

### Environment

```markdown
- **OS**: Ubuntu
- **Sage Version**: 10.0.beta9
```

### Expected Behavior

It should return the group with one element.

### Actual Behavior

```
sage: libgap.Group([])
---------------------------------------------------------------------------
GAPError                                  Traceback (most recent call last)
Cell In [54], line 1
----> 1 libgap.Group([])

File ~/sage/src/sage/libs/gap/element.pyx:2535, in sage.libs.gap.element.GapElement_Function.__call__()
   2533 try:
   2534     sig_GAP_Enter()
-> 2535     sig_on()
   2536     if n == 0:
   2537         result = CALL_0ARGS(self.value)

GAPError: Error, usage: Group(<gen>,...), Group(<gens>), Group(<gens>,<id>)
```

### Additional Information

Found when trying

```
WeylGroup(SymmetricGroup(1))
```

**Comment by fingolfin** [comment]
If you call `Group([])` it will also produce an error, and there is no alternative: you want a trivial group, sure, but which kind of trivial group? Permutation group? Fp group? Matrix group (and over which field and degree)? etc.

So the bug is in the code calling `Group` here, which does not handle the case where it is given a trivial group. One typical fix is to use the two argument version of `Group` which takes a list of generators (which may be empty) plus the identity element of the group. In pure GAP, that's how it might look like:
```gap
G := SymmetricGroup(1);
H := Group(GeneratorsOfGroup(H), One(G));
```
Hope that helps.

**Comment by DaveWitteMorris** [comment]
I think this issue should be closed, because it was fixed by PR #35730.

**Comment by fchapoton** [comment]
ok, let's close
